src/configurations/destinations/accoil_analytics/ui-config.json

```diff
@@ -0,0 +1,196 @@
+{
+  "uiConfig": {
+    "baseTemplate": [
+      {
+        "title": "Initial setup",
+        "note": "Review how this destination is set up",
+        "sections": [
+          {
+            "groups": [
+              {
+                "title": "Connection settings",
+                "note": "Update your connection settings here",
+                "icon": "settings",
+                "fields": [
+                  {
+                    "type": "textInput",
+                    "label": "API Key",
+                    "configKey": "apiKey",
+                    "regex": "^[0-9a-zA-Z_]{1,80}$",
```

Comment:
this apiKey regex is not matching with apiKey property in schema.json

Reply:
I've re-generated schema.json which has picked up the changed regex.